configs/modelbuilders/xgb_mb_cpu_config_shap.json

```
@@ -0,0 +1,309 @@
+                    "type": "regression",
+                    "n_features": 50,
+                    "n_informative": 50,
+                    "training": {
+                        "n_samples": 2000
+                    },
+                    "testing": {
+                        "n_samples": 80000
+                    }
+                }
+            ]
+        },
+        {
+            "objective": "reg:squarederror",
+            "max-depth": 14,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 4096,
+            "dataset": [
+                {
+                    "source": "synthetic",
+                    "type": "regression",
+                    "n_features": 50,
+                    "n_informative": 50,
+                    "training": {
+                        "n_samples": 4000
+                    },
+                    "testing": {
+                        "n_samples": 80000
+                    }
+                }
+            ]
+        },
+        {
+            "objective": "reg:squarederror",
+            "max-depth": 8,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 256,
+            "dataset": [
+                {
+                    "source": "synthetic",
+                    "type": "regression",
+                    "n_features": 100,
+                    "n_informative": 100,
+                    "training": {
+                        "n_samples": 1000
+                    },
+                    "testing": {
+                        "n_samples": 80000
+                    }
+                }
+            ]
+        },
+        {
+            "objective": "reg:squarederror",
+            "max-depth": 10,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 1024,
+            "dataset": [
+                {
+                    "source": "synthetic",
+                    "type": "regression",
+                    "n_features": 100,
+                    "n_informative": 100,
+                    "training": {
+                        "n_samples": 2000
+                    },
+                    "testing": {
+                        "n_samples": 80000
+                    }
+                }
+            ]
+        },
+        {
+            "objective": "reg:squarederror",
+            "max-depth": 14,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 4096,
+            "dataset": [
+                {
+                    "source": "synthetic",
+                    "type": "regression",
+                    "n_features": 100,
+                    "n_informative": 100,
+                    "training": {
+                        "n_samples": 3000
+                    },
+                    "testing": {
+                        "n_samples": 80000
+                    }
+                }
+            ]
+        }
+    ]
+}
```

Line break is required at end of file
```suggestion
}

```